README.md

```diff
@@ -1,2 +1,20 @@
 # HW01
-Edit the readme
+## Ariana F. Tortolani 
+### 07/05/2020
+
+
+## About Me
+Hi!  My name is Ariana and I am from Hope, Rhode Island.  I am an incoming Ph.D. Student at **The University of Chicago** in the *Committee on Computational Neuroscience*.  Currently, I am interested in studying motor control and proprioception.  I am interested in how these properties work together through sensori-motor integration to allow an individual to successfully perform goal-directed behaviors such as reaching out to pick up an object.
+
+I just graduated from **Boston University** this past May with my *B.A. in Neuroscience*.  During my time at BU, I worked in the [Hasselmo Lab](https://www.bu.edu/hasselmo/) and the [Theoretical Cognitive Neuroscience Lab](http://sites.bu.edu/tcn/) where I studied hippocampal time cells.  While most people are typically aware of spatial mapping (i.e. hippocampal place cells), more recent studies provide evidence in support of distinct temporal mapping in the hippocampus. This temporal organization occurs via sequences of neurons that fire at specific moments during temporally structured events. These neurons have become known as “time cells”. 
+
+Here's a photo of me:
+![Graduation photo in front of the College of Arts & Sciences at BU.](https://scontent-bos3-1.xx.fbcdn.net/v/t1.0-9/98593905_2606209996367015_9072551270958497792_n.jpg?_nc_cat=108&_nc_sid=09cbfe&_nc_ohc=5Ee6zh6dqHkAX9gCQ54&_nc_ht=scontent-bos3-1.xx&oh=4f3475759e34deb70022d20f65a4adc1&oe=5F291562)
```

Comment:
Congratulations on your graduation! I hope you had a great time despite the pandemic and took lots of pictures! 

That aside, I noticed that you wrote a caption for your image in the codes but it didn't appear when I switch to preview them in html. I wonder why is that so because I noticed the same problem with mine.

Reply:
I'm glad you noticed that, I was wondering the same thing!  When I was playing around with inserting a different image / caption in RStudio the figure caption did show up when I previewed it in html but for some reason this one was not.  I'm really not sure what the answer to this is.

I guess this also gets at maybe I should have hit commit on my code a few more times and then would have been able to compare the differences, I kept pressing save but not commit when making changes, I guess I'm just not used to that programming style yet.